napari/_app/_tests/test_app.py

```diff
@@ -0,0 +1,25 @@
+from napari._app import app
+from napari.layers import Points
+
+
+def test_app():
+    """just make sure our app model is registering menus and commands"""
+    assert app.name == 'napari'
+    assert list(app.menus)
+    assert list(app.commands)
+    # assert list(app.keybindings)  # don't have any yet
+
+
+def test_app_injection():
+    """Simple test to make sure napari namespaces are working in app injection."""
+
+    def use_points(points: 'Points'):
+        return points
+
+    p = Points()
+
+    def provide_points() -> 'Points':
+        return p
+
+    with app.injection_store.register(providers=[(provide_points,)]):
+        assert app.injection_store.inject(use_points)() is p
```

Comment:
injection returning the function makes this a little hard to parse, at least for me. Is the following equivalent? 
```suggestion
        app.injection_store.inject(use_points)
        assert use_points() is p
```

Reply:
it's not quite actually!  because you need to store the injected version of `use_points` (i.e. calling `inject` doesn't modify the object it's being called on... like most decorators, it returns a new object).  but we could do this:
```
injected_use_points = app.injection_store.inject(use_points)
assert injected_use_points() is p
```

like that better?

Reply:
(changed)

Reply:
Missed this last night, thanks for changing! I find the alternative much more readable :)